charts/opentelemetry-collector/Chart.yaml

```diff
@@ -1,6 +1,6 @@
 apiVersion: v2
 name: opentelemetry-collector
-version: 0.65.1
+version: 0.66.0
```

Comment:
It seems more like a bug fix that should go to a patch version?

Reply:
Fair, I'll update the version